stm32/unittest/README_unittest.txt

```diff
@@ -71,32 +82,13 @@ Building and Running the stm32 Unit Tests
   2018-12-29T06:52:16 INFO common.c: SRAM size: 0x30000 bytes (192 KiB), Flash: 0x100000 bytes (1024 KiB) in pages of 16384 bytes
   2018-12-29T06:52:16 INFO gdb-server.c: Chip ID is 00000413, Core ID is  2ba01477.
   2018-12-29T06:52:16 INFO gdb-server.c: Listening at *:4242...
-
-2/ The STM32 Standard Preipheral Library is required and requires
-   registration to download. Save the zip file somewhere safe, then
-   extract to a directory of your choice, for example:
-
-   $ cd /periph/lib/path
-   $ unzip /path/to/en.stm32f4_dsp_stdperiph_lib.zip
-
-   The unittest Makefile requires a path to the unzipped Standard
-   Preipheral Library, this can be set by adding a local.mak file in
-   the codec2-dev/stm32/unittest directory:
-
-   $ cd ~/codec2-dev/stm32/unittest/src
-   $ echo 'PERIPHLIBDIR = /periph/lib/path/STM32F4xx_DSP_StdPeriph_Lib_V1.8.0' > local.mak
-
-3/ Now we can build the unittests:
-
-   $ cd codec2-dev/stm32/unittest/src
-   $ make
   
```

Comment:
@donmr - so you are no longer building yr tests here?

Reply:
Correct, I am using CMake which builds in build_stm32/unittest/src (usually located in codec2/stm32).